x-pack/plugins/apm/public/components/shared/kuery_bar/index.tsx

```diff
@@ -125,7 +125,10 @@ export function KueryBar(props: { prepend?: React.ReactNode | string }) {
     }
 
     try {
-      const res = convertKueryToEsQuery(inputValue, indexPattern);
+      const res = convertKueryToEsQuery(
+        inputValue,
+        indexPattern as IndexPattern
```

Comment:
Is `as IndexPattern` really needed?

Reply:
it is, `indexPattern` is typed as `IndexPatternTitleAndFields` which `convertKueryToEsQuery` complains about.